mon-pix/app/authenticators/pole-emploi.js

```diff
@@ -2,18 +2,19 @@ import RSVP from 'rsvp';
 import { isEmpty } from '@ember/utils';
 import { inject as service } from '@ember/service';
 
-import OIDCAuthenticator from 'ember-simple-auth-oidc/authenticators/oidc';
```

Comment:
**nitpick**
Ce commit parle de corriger le logout pole emploi, mais ce qu'il fait en réalité c'est retirer le besoin de dépendre de `ember-simple-auth-oidc` pour dépendre directement de `ember-simple-auth`. C'est donc plus simple "fix". 
En tout logique, il devrait aussi arriver avant le commit qui retire la dépendance du package.json (si on voulait que les tests passent à chaque commit).

Reply:
done, nouveau nom de commit `use ember-simple-auth in authenticators and update invalidate method for logout`

Reply:
et bien d'accord pour mettre comme dernier commit la suppression du package de nos dépendances